controller/TransparencyController.js

```diff
@@ -0,0 +1,50 @@
+const { Donation } = require("../services");
+const catchAsyncError = require("../middleware/catchAsyncError");
+
+/**
+ * @description get total donations
+ * @route GET /v1/transparency
```

Comment:
Please let's update the route to be `/v1/donations/range`

Reply:
Done.